### ISSUE TITLE
jcamp metadata in PXRD view look weird

### ISSUE THREAD

**Issue by kjappelbaum** [cheminfo/eln.epfl.ch#276]
![Screen Shot 2021-03-11 at 11 53 06](https://user-images.githubusercontent.com/32935233/110776501-6c471080-8260-11eb-9fcc-d5d0cf4cf168.png)

**Comment by kjappelbaum** [comment]
starTime should not be endTime, that makes no sense. Also `object Object` is not really useful

**Comment by lpatiny** [comment]
I guess this is related to the extraction of metadata there:

https://github.com/cheminfo/xrd-analysis/blob/3d256376e18b9d894e8e1be1cb0d9b152c2a4984/src/from/parseBRML.js#L31-L60

**Comment by kjappelbaum** [comment]
yes and we should, upon importation, also put directly the anode in the json
